src/utils.js

```diff
@@ -54,24 +54,27 @@ const getCourseNumber = (courseKeyFragment) => {
   return keyParts[keyParts.length - 1];
 };
 
-const getErrorMessage = (error) => {
+const getErrorMessages = (error) => {
   if (typeof error === 'object') {
     // For form validation from DRF, comes back as an object of fields:errors
-    if (error.response && error.response.data && typeof data === 'object') {
+    if (error.response && error.response.data && typeof error.response.data === 'object') {
       return Object.keys(error.response.data).map(key => `${key}: ${error.response.data[key]}`);
     }
-    // For base JS errors check .message first before checking the response
-    return error.message || error.response.data || error.response.message;
+    // Some request responses contain a .message, as well as JS errors let's
+    // try to use the request data or message before the or base JS errors check
+    return [
+      (error.response && (error.response.data || error.response.message)) || error.message || error,
+    ];
   } else if (typeof error === 'string') {
-    return error;
+    return [error];
   }
-  return null;
+  return 'Unknown error';
```

Comment:
Should this be `['Unknown error.']` ?

Reply:
Yeah can do that, would also love other sensible recommendations for this? Should we make a best effort and stringify whatever we get?